source/mongodb/data-access.txt

```diff
@@ -0,0 +1,124 @@
+.. _client-data-access:
+
+============================
+Query Atlas from Client Apps
+============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can query data in MongoDB Atlas from client applications using standard
+MongoDB query language syntax with **MongoDB Data Access** in the Realm SDKs.
+
+The MongoDB Data Access secures data with role-based App Services rules, schemas,
+and filters. These features let construct complex client-side queries
+while your data remains secure on the server.
+
+Realm SDK Support
+-----------------
+
+The following Realm SDKs support MongoDB Data Access:
+
+- :ref:`Java SDK <java-intro>`
+- :ref:`.NET SDK <dotnet-intro>`
+- :ref:`Node.js SDK <node-intro>`
+- :ref:`React Native SDK <react-native-intro>`
+- :ref:`Swift SDK <ios-intro>`
+- :ref:`Web SDK <web-intro>`
```

Comment:
Instead of linking to intros, could we combine this with the section at the bottom of the page and link straight to the MDB pages for each SDK?